README.md

```diff
@@ -1 +1,15 @@
 # Mobile Logstash Encoder
+
+Mobile wishes to capture App, Stack and Stage using the simple-configuration AWS identity detection.
+
+This library provides an encoder extends the LogstashEncoder to set the custom fields.
+
+Add the MobileLogstash to an appender like
+
+```xml
+   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
I know it's a detail, but could the example be a rolling file? So that it becomes a no-brainer for whoever integrates it.